src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingComment.java

```diff
@@ -24,10 +24,25 @@ void method2(long ms /* we should ignore this */) {
     }
 
     /**
-     * comment with trailing space
+     * comment with trailing space.
      */
     final static public String NAME="Some Name"; // NOI18N
     final static public String NAME2="Some Name"; /*NOI18N*/
     String NAME3="Some Name"; /*NOI18N
 */
+    void method3() {
+        /* violation on this block */
+        // violation here
+    }
+
+    private static class TimerEntry {
+        /* ok */ final String operation = null;
+        /* ok */ final long start = 0L;
+    }
+
+    /**
+     * violation above this line.
+     **/
+    /* ok */ void addError() {
```

Comment:
`/* package */ void addError() { // ok`

please apply it to other comments like this.
People do this leading comment to bring attention that visibility of field is not private, it is "package"

Reply:
Done, left `method1()`  intentionally so as to make things clear `void method()` is also valid for this check.